hereditary/README.md

```diff
@@ -0,0 +1,7 @@
+# /HEREDITARY Ontology/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for the HEREDITARY Ontology. 
+This project has received funding from the HEREDITARY Project, as part of the European Union’s Horizon Europe research and innovation programme under grant agreement No GA 101137074.
+
+## Contact:
+* **Laura Menotti** (<laura.menotti@unipd.it>)
+* **Gianmaria Silvello** (<silvello@dei.unipd.it>)
```

Comment:
Please add github user ids if available.  (We use those to check who is allowed to make changes.)

Reply:
```suggestion
* **Laura Menotti** (<laura.menotti@unipd.it>) @mntlra
* **Gianmaria Silvello** (<silvello@dei.unipd.it>) @giansilv
```